### ISSUE TITLE
Accuracy drops a lot, after convert quantized onnx model to TensorRT8.2 engine.

### ISSUE THREAD

**Issue by apintec** [NVIDIA/TensorRT#1568]
### Description
Firstly, i use pytorch to train a fp32 model (with test acc 90.5%).  And then i convert the pth model to onnx model (with test acc  90.5%),  and then convert to tensorRT8.2 engine (with test acc 90.5%). The three different types of model have same acc, This is a normal phenomenon.

Secondly, i quantize the fp32 model above use  **QAT** (TensorRT Quantization Toolkit for PyTorch),  the quantized model with acc  90.15%, and then convert to onnx model (with test acc 89.8%, a little drop but acceptable). At last, i convert the quantized onnx model to TensorRT8.2 engine succesfully. **BUT** the acc of the quantized  .trt  model drop to 75.75%. 

So, I want to know why the acc droped a lot after QAT onnx model convert to tensorRT engine. and how to solve it. Thanks very much!

### Environment
TensorRT Version: 8.2.0.6
NVIDIA GPU: RTX2060
NVIDIA Driver Version: 466.63
CUDA Version: 11.1
CUDNN Version: 8.1
Operating System: windows10
Python Version: 3.7
Torch Version: 1.9.0
pytorch-quantization Version: Release 8.0

### Relevant files
I have uploaded the quantized onnx model as mentioned above and a test image. And i have resize the image to 224*224, so the only image preprocess is Normalize form [0-255] to [0-1], i.e. mean=[0.0,0.0,0.0],std=[255.0,255.0,255.0].
The inference result from onnx model and test image is: [-5.7190, 5.7229].
but after convert to tensorRT8.2 engine, the result inferred on the same image is: [3.789, -3.764].
The test results were the exact opposite！
[model&testImage.zip](https://github.com/NVIDIA/TensorRT/files/7396058/model.testImage.zip)

**Comment by apintec** [comment]
@zifeis  thansk for reply and i solved the problem by replace the activation layer from HSwish to Relu6.

**Comment by yq20212102** [comment]
学习到了 里面的激活层要还换。。。

**Comment by ynma-hanvo** [comment]
> @zifeis thansk for reply and i solved the problem by replace the activation layer from HSwish to Relu6.

gald to hear that you fixed it.  but  I wonder how did you find out this solution?
